semantic_conventions/trace/cloudevents.yaml

```
@@ -8,25 +8,25 @@ groups:
     attributes:
       - id: event_id
         type: string
+        required: always
         brief: >
           The [event_id](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#id) uniquely identifies the event.
```

```suggestion
          The [event_id](https://github.com/cloudevents/spec/blob/v1.0.2/spec.md#id) uniquely identifies the event.
```
Would be the up-to-date version (+ a few more occurrences below).

---

Good find. Fixed it everywhere.